products/public-websites/cerner/postmortem/new-facilities-considered-cerner-03-09-2021.md

```diff
@@ -0,0 +1,134 @@
+
+_Date/time of discovery_ March 8, 2021, 2:20pm EST
+
+## Impact
+
+_What was the impact on Veterans, business and team?_
+
+Any user with any of the following facilities:
+
+```
+'463', // Alaska VA
+'531', // Boise, ID
+'648', // Portland, OR
+'653', // Roseburg (Roseburg OR)
+'663', // Puget Sound (Seattle WA)
+'687', // Walla Walla, WA
+'692', // White City, OR
+```
+
+would have experienced links for those respective facilities to My VA Health instead of My Healthe Vet on the following routes:
+
+```
+https://www.va.gov/health-care/refill-track-prescriptions/
+https://www.va.gov/health-care/secure-messaging/
+https://www.va.gov/health-care/schedule-view-va-appointments/
+https://www.va.gov/health-care/view-test-and-lab-results/
+https://www.va.gov/health-care/get-medical-records/
+```
+
+from March 5, 2021 @ 11:40am EST until March 8, 2021 @ 5:55pm EST.
+
+## Root Causes
+
+_What caused this situation?_
+
+The cause of the issue (the missing feature toggles in the feature toggle network request) [was identified on March 8, 2021 @ 2:54pm EST](https://dsva.slack.com/archives/C52CL1PKQ/p1615233244248900?thread_ts=1615231277.236500&cid=C52CL1PKQ).
+
+## Resolution
+_What steps were taken to resolve the issue?_
+
+[A PR that fixed the issue passed CI and was merged @ 4:07pm EST](https://dsva.slack.com/archives/CBU0KDSB1/p1615237653334300?thread_ts=1615233939.312300&cid=CBU0KDSB1). An off-schedule [deploy to production was then initiated and completed at March 8, 2021 @ 5:55pm EST](https://dsva.slack.com/archives/CBU0KDSB1/p1615244095346700?thread_ts=1615233939.312300&cid=CBU0KDSB1).
+
+_Date/time of resolution_
+
+March 8, 2021 @ 5:55pm EST
+
+## Documentation links
+
+- [`vets-api` PR to add feature toggles](https://github.com/department-of-veterans-affairs/vets-api/pull/6045)
+- [`vets-website` PR to add Cerner facility IDs (culprit)](https://github.com/department-of-veterans-affairs/vets-website/pull/16220)
+- [Detection](https://dsva.slack.com/archives/C52CL1PKQ/p1615231277236500)
+- [`vets-website` PR to add missing feature toggles (fix)](https://github.com/department-of-veterans-affairs/vets-website/pull/16295)
+- [Off-schedule deploy](https://dsva.slack.com/archives/CBU0KDSB1/p1615233939312300)
+
+## Action Items
+
+| Description                    | Type    | Owner        | Issue # |
+| ------------------------------ | ------- | ------------ | ------- |
+| We should add a unit test [here](https://github.com/department-of-veterans-affairs/vets-website/blob/dbd95d9098ccd5acc1787ceb9ee3fc55da3e7852/src/platform/utilities/cerner/index.js#L5) to ensure that any Cerner facility ID that is added to `vets-website` has a related feature toggle [here](https://github.com/department-of-veterans-affairs/vets-website/blob/3f41da4363074b54dc3efab7baff0f4898bb47a4/src/platform/utilities/feature-toggles/featureFlagNames.js#L67) | Frontend | Public Websites | Ticket not created yet |
```

Comment:
@brianalloyd we may need a ticket for this

Reply:
@kelsonic Ticket #21235 to address the above.